docs/index.rst

```diff
@@ -36,6 +36,12 @@
    notes/dask.rst
    notes/ml.rst
 
+.. toctree::
+   :maxdepth: 6
+   :caption: Examples
+   
+   notes/distibuted_learning.rst
```

Comment:
Its ok to put it under the "Thirt party integration" chapter, IMO.

Just after the `ml.rst`.

Reply:
I would like to have an independent example directory where real-world examples are placed. While the 3rd party integration directory focuses on "Integration". Do you think it's OK?